pyflask/manageNeuroconv/manage_neuroconv.py

```diff
@@ -582,11 +582,17 @@ def upload_folder_to_dandi(
     cleanup: Optional[bool] = None,
     number_of_jobs: Optional[int] = None,
     number_of_threads: Optional[int] = None,
+    ignore_cache: Optional[bool] = None,
 ):
     from neuroconv.tools.data_transfers import automatic_dandi_upload
 
     os.environ["DANDI_API_KEY"] = api_key  # Update API Key
 
+    if ignore_cache:
+        os.environ["DANDI_CACHE"] = "ignore"
+    else:
+        os.environ.pop("DANDI_CACHE", None)
```

Comment:
Tracing all the way down to the cacher they use, https://github.com/con/fscacher/blob/6ccd52f1845c02cce99cbb4b849c0debf2cdc76a/src/fscacher/cache.py#L23

The proper way I think would be to set this to just an empty string. That's without getting complicated with actually modifying the existence status of the environment variable; i.e., `pop` removes it from the mapping exposed to this Python process but doesn't unset it in the parent

```suggestion
    else:
        os.environ["DANDI_CACHE"] = ""
```